Makefile

```diff
@@ -329,11 +332,15 @@ ci-check: clean-downloadcache deps fast-babel check
 bumpversion: deps
 	bin/bumpversion $(VPART)
 
+.PHONY: version
+version:
+	echo "'version: $(CURRENT_VERSION)';\n'commit: $(CURRENT_COMMIT)';" > $(GUIBUILD)/app/version.js
```

Comment:
Because the combo loader and build process handle JS just fine but balk at JSON. I spent some time trying to get JSON working but just threw my hands up in the end and switched back to JS. I remember `'foo bar'` used to be valid JS, which is why the strict mode invocation took the form it did (older JS engines wouldn't balk, ignoring it instead), but it seems ECMAScript 5 takes a more stringent view of bare strings in JS files. Meh.